build/klh10/config.203

```diff
@@ -993,10 +998,6 @@ DEFOPT IMPUS==100.	; IMP net host number (old-style) (100 octal)
 DEFOPT IMPUS3==<IPADDR %IP%>	; IP address
 DEFOPT NM%IMP==<IPADDR %NETMASK%>	; Subnet mask
 DEFOPT PKTTRC==-1	;Packet tracing code enabled
-DEFOPT CHAOSP==%CHAOSP% ;Has CHAOS net
```

Comment:
So we can no longer build DB for a specified chaosnet address and only for the local chaosnet address?  This seems lame to me.  If I don't want to use a local chaosnet, but rather have my own bridge or want to use the global router, I may have a chaosnet address and want to use it.  It appears that now I cannot.

Reply:
That's right, *DB* ITS will be hardcoded to 177002.  The upside of this is that all beginner users can get started running ITS with a local Chaos network without any modifications.  Users that want their own Chaosnet address and connect to the global network can still do so by setting up another ITS.